core/src/dird/ua_prune.cc

```diff
@@ -855,15 +854,28 @@ bool PruneVolume(UaContext* ua, MediaDbRecord* mr)
   if (bstrcmp(mr->VolStatus, "Full") || bstrcmp(mr->VolStatus, "Used")) {
     Dmsg2(050, "get prune list MediaId=%d Volume %s\n", (int)mr->MediaId,
           mr->VolumeName);
-    count = GetPruneListForVolume(ua, mr, &del);
-    Dmsg1(050, "Num pruned = %d\n", count);
-    if (count != 0) { PurgeJobListFromCatalog(ua, del); }
-    ok = IsVolumePurged(ua, mr);
+
+
+    int NumJobsToBePruned = GetPruneListForVolume(ua, mr, &del);
+    Jmsg(ua->jcr, M_INFO, 0,
+         _("Pruning volume %s: %d Jobs have expired and can be pruned.\n"),
+         mr->VolumeName, NumJobsToBePruned);
+    Dmsg1(050, "Num pruned = %d\n", NumJobsToBePruned);
+    if (NumJobsToBePruned != 0) { PurgeJobListFromCatalog(ua, del); }
+    VolumeIsNowEmtpy = IsVolumePurged(ua, mr);
+
+    if (!VolumeIsNowEmtpy) {
+      Jmsg(ua->jcr, M_INFO, 0,
+           _("Volume \"%s\" still contains jobs after pruning.\n"));
```

Comment:
How did I miss this bug?
This is btw. why we want a modern formatting library that would have detected this at compile-time...